src/gpu/intel/jit/v2/conv/kernel_desc.cpp

```diff
@@ -38,6 +38,58 @@ namespace jit {
 namespace v2 {
 namespace conv {
 
+void specialization_t::specialize(const problem_t &prb) {
+    if (min_dims) {
+        auto t = min_dims_tile(prb);
+        for (auto &d : t) {
+            ir_assert(!spec_tile.has(d) || spec_tile[d] == t[d]);
+            spec_tile[d] = t[d];
+        }
+        min_dims = false;
+    }
+    if (max) {
```

Comment:
I think it is better to keep `min_dims` and `max` as an enum,  allowing mixing of strategies just seems like it can introduce complexities, for example `max=true` and `min={true,false}` are effectively the same. If we stick with an enum, we can simplify this to something like:
```
        auto t = get_dims_tile(prb, mode)
        for (auto &d : t) {
            ir_assert(!spec_tile.has(d) || spec_tile[d] == t[d]);
            spec_tile[d] = t[d];
        }
        mode=none;
```

Reply:
Agree, switched back to enum, named `specialization_mode_t`.